gradle-guide/src/main/java/com/palantir/gradle/guide/GradleGuidePlugin.java

```diff
@@ -28,7 +28,8 @@ public class GradleGuidePlugin implements Plugin<Project> {
             "ConfigurationAvoidanceRegistration",
             "ProviderGet",
             "NonAbstractGradleType",
-            "IllegalMethodCalledDuringTaskExecution");
+            "IllegalMethodCalledDuringTaskExecution",
+            "UseGradleLoggerInGradlePlugins");
```

Comment:
This doesn't actually have any fixes, but I guess no harm in putting here for now. The ideal would be a "include everything in package" option in the suppressible-error-prone extension.